database/seeds/PlaceTagTableSeeder.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Hedonist\Entities\Place\Place;
+use Hedonist\Entities\Place\PlaceTag;
+use Illuminate\Support\Facades\DB;
+
+class PlaceTagTableSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $places = Place::all();
+
+        foreach ($places as $place) {
+            $getTagId = DB::table('place_category_place_tag')
+                ->where('place_category_id', $place->category_id)
+                ->pluck('place_tag_id')
+                ->toArray();
+
+            PlaceTag::create([
```

Comment:
Better to use a bit another algorithm here. Fetch places and tags. And then attach $place->tags()->saveMany($someTags) where $someTags is 3 to 5 random tags which belong to one category.